### PR TITLE
Fix actions which operate with tags on AWS resources

### PR DESCRIPTION
**Pull request by vutny**
I have found few issues with actions which intend to set tags in AWS (they have immutable parameter `action: create_tags`).
* Action has `Tags` parameter of `array` type. It seems that such parameter should be passed directly to `boto3` fuction. For example: `aws.ec2_create_tags`.
* Action has `tags` (lowercase) parameter of type `array`. Similar to `Tags` above, it is expected that they should satisfy Boto call. For example `aws.discovery_create_tags`.
* Action has `tags` parameter of type `string`. Format is: `key1=value1,key2=value2,...`. For example: `aws.vpc_create_tags`.

This PR fixes `run.py` action runner script by transforming `tags` kwarg to a list of dicts if it is a string, otherwise pass `Tags` or `tags` untouched straight to Boto3 action.

Affected actions are:
* aws.ec2_create_tags
* aws.efs_create_tags
* aws.discovery_create_tags
* aws.redshift_create_tags
* aws.vpc_create_tags
* aws.workspaces_create_tags

@LindsayHill Could you please review my PR? Thank you.

### CODE REVIEW

**Comment by vutny** [comment]
pylint error on CircleCI seems unrelated to my changes, since `actions/lib/action.py` remains unchanged.

File: actions/run.py
Comment (bigmstone) [inline actions/run.py]: Let's use `pop` here since the next line of code is to delete action and that can all be accomplished w/ a single call to `pop`.

**Review by bigmstone — APPROVED** [review]
These changes look good to me. I'm fixing the pylint issue up in a separate PR. Please update `master` once it's merged and we can merge this once it passes tests.

**Comment by bigmstone** [comment]
@vutny `master` is updated now. If you'll update your branch I'll get this merged.

File: actions/run.py
Comment (vutny) [reply actions/run.py]: @bigmstone Thank you for your review! Sure thing, this could be "popped", I was just afraid to touch too much lines as first-time contributor here :smile:

**Comment by vutny** [comment]
Rebased against latest master and all is green now! Thanks @bigmstone

**Comment by vutny** [comment]
Resigned my commit with GnuPG key.

**Pull request merged**
